web/src/modules/feedback/components/FeedbackBox.js

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react'
 
-import { withTranslation, TFunction } from 'react-i18next'
+import { withTranslation, type TFunction } from 'react-i18next'
```

Comment:
Is it `import type` of `typeof` now?

Reply:
Not sure I understand your comment. `import type` has always been the right way to import types.